cli/tests/unit/url_search_params_test.ts

```diff
@@ -49,10 +49,39 @@ unitTest(function urlSearchParamsInitString(): void {
 });
 
 unitTest(function urlSearchParamsInitStringWithPlusCharacter(): void {
-  const init = "q=a+b";
-  const searchParams = new URLSearchParams(init);
-  assertEquals(searchParams.toString(), init);
-  assertEquals(searchParams.get("q"), "a b");
+  let params = new URLSearchParams("q=a+b");
+  assertEquals(params.toString(), "q=a+b");
+  assertEquals(params.get("q"), "a b");
+
+  params = new URLSearchParams("q=a+b+c");
+  assertEquals(params.toString(), "q=a+b+c");
+  assertEquals(params.get("q"), "a b c");
+});
+
+unitTest(function urlSearchParamsInitStringWithMalformedParams(): void {
+  // These test cases are copied from Web Platform Tests
+  // https://github.com/web-platform-tests/wpt/blob/54c6d64/url/urlsearchparams-constructor.any.js#L60-L80
```

Comment:
Not that it is required for this PR, but why would we not want all of the WPT tests here?

Reply:
Sounds a good idea. I'll try to add missing test cases from WPT in another branch.